### DISCUSSION TITLE
How to access coder cloud through custom domain

### DISCUSSION THREAD

**Discussion by joshleblanc** [coder/code-server#2521] [Q&A]
Set up the coder cloud link (`code-server --link`), and I wanted to set up a custom domain to point to it. 

I set up a CNAME record that points to the coder cloud URL, but it doesn't appear to resolve. Is this intentionally blocked, or is there another way to do it?

**Comment by code-asher** [comment]
The host header is used to determine where to proxy the request but with
a CNAME I believe the browser sends the CNAME host and not the host it
points to so the proxy has no idea where to route the request.

A reverse proxy would probably work as long as it set the right host
header.

**Reply by joshleblanc** [reply]
I realized the SSL cert still wouldn't be for the custom domain anyway, so it was kind of a lost cause to begin with. 

I opted for just a redirect, which works fine.
